lib/hubspot/company.rb

```diff
@@ -33,11 +34,11 @@ def all(opts = {})
     def search_domain(domain, opts = {})
       Hubspot::PagedCollection.new(opts) do |options, offset, limit|
         request = {
-          "limit" => limit,
-          "requestOptions" => options,
+          "limit" => 2,
```

Comment:
why did we limit to only 2?

Reply:
https://legacydocs.hubspot.com/docs/methods/companies/search_companies_by_domain

@lachi87 as per these docs I have configured the limit to 2